src/group_theory/schur_zassenhaus.lean

```diff
@@ -162,21 +162,206 @@ begin
     apply_instance },
 end
 
-/-- **Schur-Zassenhaus** for abelian normal subgroups:
-  If `H : subgroup G` is abelian, normal, and has order coprime to its index, then there exists
-  a subgroup `K` which is a (right) complement of `H`. -/
-theorem exists_right_complement'_of_coprime [fintype G] [H.normal]
+-- Do not use this lemma: It is made made obsolete by `exists_right_complement'_of_coprime` -/
```

Comment:
maybe private lemmata in this file instead of these "do not use" comments? also this specific one isn't a doc comment